tests/devices/test_default_qutrit.py

```diff
@@ -260,6 +260,277 @@ def test_apply_rotations_one_wire(self, qutrit_device_1_wire, tol):
     # TODO: Add tests for state preperation ops after they're implemented
 
 
+class TestExpval:
+    """Tests that expectation values are properly calculated or that the proper errors are raised."""
+
+    # TODO: Add test for expval of non-parametrized observables
+
+    @pytest.mark.parametrize(
+        "operation,input,expected_output,par",
+        [
+            (qml.THermitian, [1, 0, 0], 1, [[1, 1j, 0], [-1j, 1, 0], [0, 0, 1]]),
+            (qml.THermitian, [0, 1, 0], 1, [[1, 1j, 0], [-1j, 1, 0], [0, 0, 1]]),
+            (
+                qml.THermitian,
+                [1 / math.sqrt(3), -1 / math.sqrt(3), 1j / math.sqrt(3)],
+                1,
```

Comment:
It looks like all the expected outputs are 1, perhaps include some test cases where this varies?